iconservice/icon_service_engine.py

```diff
@@ -1269,7 +1272,11 @@ def _handle_icx_call(self,
             elif self._check_prep_process(params):
                 return context.engine.prep.query(context, data)
             elif self._check_debug_process(params):
-                return self._handle_get_iiss_info(context, data)
+                method_name: str = data["method"]
+                if method_name == "getIISSInfo":
+                    return self._handle_get_iiss_info(context, data)
+                elif method_name == "getServiceConfig":
+                    return self._handle_get_service_config(context, data)
```

Comment:
This function must be available although context.revision is less than Revision.IISS.value.